corehq/apps/sms/migrations/0026_add_messagingsubevent_case_id_index.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-10-13 10:50
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0025_add_custom_metadata'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='StartEnterpriseBackend',
```

Comment:
@gcapalbo I think this might've gotten pulled in from other changes you had locally.

Reply:
nope, django added it with makemigrations. it's basically a no-op migration that was never added when the start enterprises backend was added (which resides in a different app).

it's an unfortunate consequence of having proxy models in django - it doesn't change the db in any way but it has to keep track of it since SQLMobileBackend (the proxy model) resides in the sms app.